internal/storage/dbmodel/binding.go

```diff
@@ -15,3 +15,7 @@ type BindingDTO struct {
 	ExpirationSeconds int64
 	CreatedBy         string
 }
+
+type BindingStats struct {
```

Comment:
Why do we use the same name `BindingStats` for two different types?

Reply:
let's use 1 type